docs/guides/server/bootstrap-admin-recovery.adoc

```diff
@@ -0,0 +1,107 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Admin bootstrap and recovery"
+summary="Learn how to bootstrap and recover admin account.">
+
+== A temporary admin account
+
+A user or service admin account created using one of the methods described below is *temporary*. This means the account should exist only for the duration necessary to perform operations needed to gain permanent and more secure admin access. Various UI/UX elements, such as the Administration Console warning banner, labels, and log messages, will indicate to a {project_name} administrator that the account is temporary.
+
+== Bootstrapping a temporary admin account at {project_name} startup
+
+{project_name} `start` and `start-dev` commands support options for bootstrapping both temporary admin users and admin service accounts. These options are standard configuration options, so they can be specified in any of the https://www.keycloak.org/server/configuration#_configuring_sources_for_keycloak[configuration sources] such as environment variables or CLI parameters. For instance, the following examples demonstrate how to use the `start` and `start-dev` commands with CLI parameters to bootstrap a temporary admin user and an admin service account, respectively:
+
+<@kc.start parameters="--bootstrap-admin-username tmpadm --bootstrap-admin-password pass"/>
+
+<@kc.startdev parameters="--bootstrap-admin-client-id tmpadm --bootstrap-admin-client-secret secret"/>
+
+The username or client ID values can be omitted; see the <<Default values>> section below for more information.
+
+The purpose of these options is solely for bootstrapping temporary admin accounts. These accounts will be created only during the initial start of the {project_name} server when the master realm doesn't exist yet. The accounts are always created in the master realm. For recovering lost admin access, use the dedicated command described in the sections below.
+
+== Bootstrapping an admin user or service account using the dedicated command
+
+The `bootstrap-admin` command can be executed even before the first-ever start of {project_name}. Bear in mind that the server needs to be stopped prior to using this command. Its execution will trigger the creation of the initial master realm, and as a result, the startup options to bootstrap the admin user and service account will be ignored later when the server is started for the first time.
```

Comment:
How does this work with Keycloak cluster? Do all nodes need to be stopped before this procedure is executed?

Reply:
Yes, all nodes need to be stopped:
https://github.com/keycloak/keycloak/blob/1e14b310669126d2dba3d20c902054f3f253ca61/docs/guides/server/bootstrap-admin-recovery.adoc?plain=1#L27

Maybe we could more emphasis on it, make it a warn box or something.

Reply:
For example, changing it to "all {project_name} nodes"?

Reply:
Addressed as per suggestion.

Reply:
Thank you @Pepo48